nebula-logger/core/main/configuration/classes/LoggerParameter.cls

```diff
@@ -369,6 +369,21 @@ public class LoggerParameter {
         private set;
     }
 
+    /**
+     * @description Indicates if Nebula Logger will store the transaction heap limits on `LogEntry__c`, retrieved from the class `System.Limits`.
+     *              Controlled by the custom metadata record `LoggerParameter.StoreApexHeapSizeLimit`, or `true` as the default.
+     *              Relies on `LoggerParameter.StoreTransactionLimits` to be true, as well.
+     */
+    public static final Boolean STORE_HEAP_SIZE_LIMITS {
+        get {
+            if (STORE_HEAP_SIZE_LIMITS == null) {
+                STORE_HEAP_SIZE_LIMITS = getBoolean('StoreApexHeapSizeLimit', true);
```

Comment:
It looks like the CMDT dev name is singular ("limit") but the constant name is plural ("limits"). Thoughts on making both use singular ("limit")?